### PR TITLE
Django Field setting null=True checks

### PR DESCRIPTION
**Pull request by minusworld**
Three checks:
- per [documentation](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.Field.null), discourage null=True in string fields
- per [documentation](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.Field.null), require null=True when blank=True and unique=True in string fields
- per check recommendation by Django creator, require null=True when blank=True on non-text fields

### CODE REVIEW

**Comment by DrewDennison** [comment]
another one @jacobian

File: python/django/string-field-null-checks-import.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.db.models import Model, CharField, IntegerField
+
+class FakeModel(Model):
```
Comment (nbrahms) [inline python/django/string-field-null-checks-import.py]: I'd expect tests for `TextField` too

File: python/django/string-field-null-checks.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+rules:
+  - id: no-null-string-field
+    patterns:
```
Comment (nbrahms) [inline python/django/string-field-null-checks.yml]: Do you want to add `pattern-inside` and assignment checks here?

Comment (minusworld) [reply python/django/string-field-null-checks.yml]: 👍

File: python/django/string-field-null-checks-import.py (hunk 1 above)
Comment (minusworld) [reply python/django/string-field-null-checks-import.py]: 👍 added

**Pull request merged**